toolchain/sem_ir/inst_namer.cpp

```diff
@@ -1212,6 +1212,20 @@ auto InstNamer::NamingContext::NameInst() -> void {
       AddInstName("str");
       return;
     }
+    case CARBON_KIND(StructType inst): {
```

Comment:
Is this just an unrelated code move? I'm a little surprised it's in this PR, doesn't look like you're changing anything or the formatter in general...

Reply:
Oh, yeah I had added StructLiteral here, and noticed it was no sorted. Then I removed it, and we're left with the sorting fix. I guess I will leave it.